autotest/utils/pipeline_chat.py

```diff
@@ -16,6 +18,7 @@ def run_pipeline_chat_test(config,
                            cases_info,
                            model_case,
                            type,
+                           worker_id: str = '',
```

Comment:
It seems "worker_id" is not used in this function

Reply:
Pipeline testcase use subprocess so I need save result and assert the result. The worker_id used in log file name